okrs.md

```diff
@@ -0,0 +1,45 @@
+# Objectives and Key Results (OKRs)
+
+Hypha uses Objectives and Key Results for planning and setting collective goals.
+We currently define OKRs at the working group level and update them quarterly in our [**OKR tracking sheet**](https://docs.google.com/spreadsheets/d/1hMFS3IhzZOFQA-yjt7lFcoVO0Ry6erFEH7D1NswvDtA/edit). 
+
+## About OKRs
+
+_Adapted from [Trello blog](https://blog.trello.com/okrs-set-achieve-track-trello)._
+
+**OKR stands for Objectives and Key Results.** The Objectives tend to be the desired outcomes that you want, and the Key Results are the measurable ways you know you’re on track to reach them.
+
+An **Objective** helps answer the question, _“Where do I want to go?”_  
+A **Key Result** helps answer, _“How will I know I’m getting there?”_
+
+## Objectives for 2020 Q1/2 
+
+_See the objectives alongisde our key results in our [**OKR tracking sheet**](https://docs.google.com/spreadsheets/d/1hMFS3IhzZOFQA-yjt7lFcoVO0Ry6erFEH7D1NswvDtA/edit)._
+
+**Business Planning WG**:  
+O1. Build our engagement strategy  
+O2. Develop our leads and opportunities pipeline  
+O3. Gain a better sense of work that people want to do (cross-cut with Infra WG)  
+O4. Assess our financial sustainability (as “capacity to endure”) (cross-cut with Finance WG)  
+O5. Shorten our lead-to-proposal time (i.e. identifying an opportunity > getting member buy-in > submitting a proposal)  
+
+**Finance WG**:  
+O1. Define robust processes to support financial activities of the co-operative  
+O2. Have well-defined roles and responsibilities to ensure smooth financial activities  
+O3. Effectively communicate our financial outlook to members  
+
+**Governance WG**:  
+O1. Formalize a governance structure that meets legal requirements  
+O2. Build [governance] relationships with "neighbours" (cross-cut with Biz Dev/Ops WGs)  
```

Comment:
Nitpick: In other places we call the business wg "Business Planning" and not "Biz Dev"

Reply:
Yes I've noticed this, however we also use BizDev a bunch of places as a shorthand (e.g., calendar). Happy to change here